PYME/Analysis/tile_pyramid.py

```diff
@@ -254,7 +254,20 @@ def flush(self):
     def __del__(self):
         self._cur.close()
         self._conn.close()
-    
+
+
+def tile_init_ref(base_dir, map_type_tile):
```

Comment:
```
TILEIO_EXT = {
    '.pzf': PZFTileIO,
    '.npy': NumpyTileIO,
    '.db': SqliteTileIO,
}

def get_tileio_backend(base_directory):
    """ find TileIO backend for a given ImagePyramid
    
    Parameters
    ----------
    base_directory : str
        root directory of an ImagePyramid instance
    
    Returns
    -------
    class
        which TileIO derived class the ImagePyramid can be
        built with.
    """
    for root, dirs, files in os.walk(base_dir):
        for f in files:
            if os.path.splitext(f)[-1] in TILEIO_EXT.keys():
                return TILEIO_EXT[file_extension]
    raise IOError("No files found for loading ImagePyramid.")
```

Reply:
Makes sense to me. I've included this in the next commit.